docs/package-lock.json

```diff
@@ -4897,11 +4897,12 @@
       }
     },
     "node_modules/braces": {
-      "version": "3.0.2",
```

Comment:
Please take this file change out of this commit/PR.

Reply:
I'll try👍